AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -137,6 +150,25 @@ + (instancetype)layoutWithLayout:(ASLayout *)layout position:(CGPoint)position
                             sublayouts:layout.sublayouts];
 }
 
+#pragma mark - Sublayout Elements Caching
+
+- (void)setCacheSublayoutLayoutElements:(BOOL)cacheSublayoutLayoutElements
+{
+  _cacheSublayoutLayoutElements = cacheSublayoutLayoutElements;
```

Comment:
It's probably not crucial, but we may want to short-circuit if the same value is set.